hub/util/load.py

```diff
@@ -0,0 +1,133 @@
+import re
+import numpy as np
+import pathlib
+import exiftool  # type: ignore
+from typing import Callable, Dict, List, Union
+from hub.util.exceptions import (
+    HubAutoUnsupportedFileExtensionError,
+    SampleCorruptedError,
+    ImageReadError,
+)
+from PIL import Image  # type: ignore
+
+
+IMAGE_SUFFIXES: List[str] = [".jpeg", ".jpg", ".png"]
+SUPPORTED_SUFFIXES: List[str] = IMAGE_SUFFIXES
+
+
+def _load_image(image_path: Union[str, pathlib.Path]) -> np.ndarray:
+    img = Image.open(image_path)
+    return np.array(img)
+
+
+def load(path: Union[str, pathlib.Path], symbolic=False) -> Union[Callable, np.ndarray]:
```

Comment:
Docstring? I have no idea what this function does. Also, it appears to be untested.